release-notes/3.4.0.md

```diff
@@ -0,0 +1,133 @@
+# 3.4.0 Release Notes
+
+- [Upgrade notes](#upgrade-notes)
+- [Scale to support more users](#scale-to-support-more-users)
+- [Customizable branding](#customizable-branding)
+- [Improved performance loading Contacts tab](#improved-performance-loading-contacts-tab)
+- [And more...](#and-more)
+
+## Upgrade notes
+
+### Breaking changes
+
+There are no breaking changes when upgrading from `3.3.x`.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Scale to support more users
+
+In previous versions users who had an internet connection would maintain a continuous request to the server waiting for any relevant database changes. For projects with thousands of users this caused the server to use a lot of memory to keep track of all of those requests.
+
+This change removes the continuous replication in favor of making a short lived request when;
+
+- five minutes has passed since the last replication attempt,
+- the user creates or updates a document,
+- we detect the user has come back online, or
+- the user clicks the new "Sync now" button.
+
+[medic#4805](https://github.com/medic/medic/issues/4805) and [medic#3976](https://github.com/medic/medic/issues/3976)
+
+![Screenshots](images/3.4.0-sync-now.png)
+
+## Customizable branding
+
+You can now configure the webapp logo, title, and favicon to use your project's branding [medic#4849](https://github.com/medic/medic/issues/4849). You can also add logos of your partners to show on the About page [medic#4850](https://github.com/medic/medic/issues/4850).
```

Comment:
I think it would be useful to let people know how to use it. I was going to link to [the branding documentation](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#branding), but given that it is just a line long we can perhaps include here.
 
```suggestion
You can now configure the webapp logo, title, and favicon to use your project's branding [medic#4849](https://github.com/medic/medic/issues/4849). You can also add logos of your partners to show on the About page [medic#4850](https://github.com/medic/medic/issues/4850). These can be set in the Admin console by going to the Branding tab in the Images section.
```

Reply:
Thank! I've linked to the documentation instead and will add some more info there soon, for example, how to do it with medic-conf.